scalameta/scalameta/shared/src/test/scala/scala/meta/tests/prettyprinters/PublicSuite.scala

```diff
@@ -194,6 +190,10 @@ class PublicSuite extends FunSuite {
     assert(lhs.pos.toString === """[0..3) in Input.String("foo + bar")""")
   }
 
+  test("scala.meta.io.AbsolutePath.toString") {
+    // n/a
```

Comment:
I thought about it but left it out because the test would be equivalent to the implementation.

I've added a prettyprinter test.